### DISCUSSION TITLE
Ant Media Server alters the quality of the video

### DISCUSSION THREAD

**Discussion by ugacomp** [ant-media/Ant-Media-Server#6818] [FAQs]
Hello!
I deployed Ant Media Server for a client but I noticed something about the quality of the video. I have a full HD video uploaded to Ant Media as a VOD and streaming to YouTube. Here is the sample stream link:

**ANT MEDIA STREAM**: [https://www.youtube.com/live/QdWL7GVRNBM](https://www.youtube.com/live/QdWL7GVRNBM)

I am also streaming the same video using upstream.io platform, broadcasting it to YouTube. Here is the link to the video:

**UPSTREAM.IO**: [https://www.youtube.com/live/-S-htHEiOx8](https://www.youtube.com/live/-S-htHEiOx8)

Now, you will notice that the Ant Media stream video quality is lower than the Upstream.io Video quality despite using the same video to stream. Ant Media darkens the video contrast (brighteness), altering its original quality

I am running the latest Ant Media edition. Please advise me on this issue.

CC: @yashtandon113 
CC: @muratugureminoglu 
CC: @asharamseervi

**Comment by amarantmeida** [comment]
@ugacompIT Thank you for your query. While the difference isn’t immediately apparent to me, we’d like to understand your setup better to investigate further. Typically, Ant Media Server doesn’t alter the video unless ABR (Adaptive Bitrate) is enabled on the server. Could you please confirm if ABR was used during this test?

**Reply by elijahwere** [reply]
There is no ABR configured. The client is using VoD and playlist

**Comment by lastpeony** [comment]
ant media video is unavailable

**Reply by dlattig** [reply]
Here are the two videos: 

Ant Media: https://www.youtube.com/live/Un0Rc3qBHZY

Upstream: https://www.youtube.com/live/-S-htHEiOx8

The quality difference is most noticeable in the flicker of the fire. The Ant Media one is laggy and appears to be dropping frames.
